packages/html-reporter/src/index.tsx

```diff
@@ -46,10 +47,27 @@ class ZipReport implements LoadedReport {
   private _json!: HTMLReport;
 
   async load() {
-    const zipReader = new zipjs.ZipReader(new zipjs.Data64URIReader((window as any).playwrightReportBase64), { useWebWorkers: false }) as zip.ZipReader;
-    for (const entry of await zipReader.getEntries())
-      this._entries.set(entry.filename, entry);
-    this._json = await this.entry('report.json') as HTMLReport;
+    const metadata = (window as any).playwrightMetadata;
+    if (metadata.shard) {
+      const reports: HTMLReport[] = [];
+      for (let index = 1; index <= metadata.shard.total; index += 1) {
+        try {
+          const zipReader = new zipjs.ZipReader(new zipjs.HttpReader(`/report/report-${index}.zip`), { useWebWorkers: false }) as zip.ZipReader;
```

Comment:
Let's keep those files siblings in order to be less intrusive, hard-code less names. So it'll be
```js
new zipjs.HttpReader(`report-${index}.zip`)
```